### PR TITLE
[DDCE-631]:§3 P1 - What governing document was used to set up and run…

### PR DESCRIPTION
**Pull request by anburoach**
# DDCE-631-§3 P1 - What governing document was used to set up and run your charity?

# New feature

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

# PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???

# Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

# Checklist PR Reviewer
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [x]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/controllers/regulatorsAndDocuments/SelectGoverningDocumentController.scala [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  def onSubmit(mode: Mode): Action[AnyContent] = (identify andThen getData andThen requireData).async { implicit request =>
+
+    form.bindFromRequest().fold(
+      formWithErrors =>
+        Future.successful(BadRequest(view(formWithErrors, mode))),
+
+      value =>
+        for {
+          updatedAnswers <- Future.fromTry(request.userAnswers.set(SelectGoverningDocumentPage, value).flatMap(_.set(Section2Page, false)))
```
Comment (robert-webb-hmrc) [inline app/controllers/regulatorsAndDocuments/SelectGoverningDocumentController.scala:67]: Based on what we talked about in the last refinement, should this be section 3?

File: app/views/Index.scala.html [hunk 2]
```diff
@@ -62,7 +62,7 @@ <h2 class="app-task-list__section">
                 </li>
                 <li class="app-task-list__item">
                   <span class="app-task-list__task-name">
-                    <a href="@{controllers.routes.IndexController.onPageLoad().url}"aria-describedby="governing-doc-info-completed">
+                    <a href="@{controllers.regulatorsAndDocuments.routes.SelectGoverningDocumentController.onPageLoad(NormalMode).url}"aria-describedby="governing-doc-info-completed">
                       @messages("index.section2.spoke2.label")
```
Comment (robert-webb-hmrc) [inline app/views/Index.scala.html:66]: Same here

File: app/controllers/regulatorsAndDocuments/SelectGoverningDocumentController.scala (hunk 1 above)
Comment (anburoach) [reply app/controllers/regulatorsAndDocuments/SelectGoverningDocumentController.scala:67]: yes.. Part-2.. Section-3

File: app/views/Index.scala.html (hunk 2 above)
Comment (anburoach) [reply app/views/Index.scala.html:66]: This is temporary... we will have ticket to refactor task list codes.... that time everything will be normalised

**Pull request merged**